src/app/blog/data.ts

```diff
@@ -30,7 +45,7 @@ export const allBlogs = [
     excerpt:
       "Discover how the View Transitions API enhances web experiences with seamless animations and smooth page transitions, improving user engagement and performance.",
     description:
-      "Discover how the View Transitions API enhances web experiences with seamless animations and smooth page transitions, improving user engagement and performance.",
+      "Learn how the View Transitions API enhances web experiences.",
```

Comment:
any reason for making this shorter, I kept the description length based on the seo metadata guidelines